src/test/java/com/zufar/onlinestore/security/jwt/JwtTokenFromAuthHeaderExtractorTest.java

```diff
@@ -0,0 +1,79 @@
+package com.zufar.onlinestore.security.jwt;
+
+import com.zufar.onlinestore.security.exception.AbsentBearerHeaderException;
+import jakarta.servlet.http.HttpServletRequest;
+import org.instancio.Instancio;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.test.util.ReflectionTestUtils;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class JwtTokenFromAuthHeaderExtractorTest {
+    @InjectMocks
+    private JwtTokenFromAuthHeaderExtractor jwtTokenFromAuthHeaderExtractor;
+    @Mock
+    private HttpServletRequest request;
+    private String jwtHttpRequestHeader = "Authorization";
```

Comment:
You can replace it through `HttpHeaders` class from Spring

Reply:
Fixed